tools/cli/NearObjectCli.cxx

```diff
@@ -79,6 +79,12 @@ NearObjectCli::GetRangeStopApp() noexcept
     return *m_rangeStopApp;
 }
 
+CLI::App&
+NearObjectCli::GetRawDeviceResetApp() noexcept
+{
+    return *m_rawDeviceResetApp;
+}
+
```

Comment:
For individual commands that are terminal (ie. don't have sub-commands), then I don't think there's any need to save away the `CLI::App` instance pointer. The `AddSubcommandUwbRawDeviceReset` function can be modified to throw if there's an error, or, have the caller check the pointer for non-`nullptr` and handle the failure appropriately (exiting the program would be fine). 

So, you can remove this function and the `m_rawDeviceResetApp` member. Same goes for `m_rawGetDeviceInfoApp` for the `devicegetinfo` command, I should have called this out on the prior PR.

Reply:
Good point, I'll remove these